.github/actions/setup-env/action.yaml

```diff
@@ -1,6 +1,12 @@
 name: Set up environment
 description: Set up a conda-recipe-manager environment
 
+inputs:
+  python-version:
+    description: "Version of python to use in the conda environment."
+    required: false
+    default: "3.11"
+
 runs:
   using: "composite"
   steps:
```

Comment:
This shouldn't use setup-python, but instead of setup-miniconda to make sure we have a consistent Python build in use. In fact, you'll likely be able to get rid of this setup-env workflow completely since setup-miniconda implements the steps except for the `pip install -e .` call.

Reply:
I setup-python for setup-miniconda. I was hopping to get some more of the boiler-plate out of the way, but my experiments did not pan out. The point of this action is to reduce duplicated start-up work across all the workflows, so I'm okay keeping it. It makes it easier if I have to change the setup steps in the future.